### ISSUE TITLE
CountMinSketch remove?

### ISSUE THREAD

**Issue by wv0m56** [tylertreat/BoomFilters#26]
Shouldn't there be a Remove / TestAndRemove API in CountMinSketch?

I'm willing to to submit a PR, just want to make sure first that there's no hole in my reasoning.

Pseudo code:

```
Compute k hashes h1, h2, ... hk
let count1, count2, ... countk = the cells h1...hk point to
min := Minimum(count1 ... countk)
if min > 0
    conclude element exists
    decrement count1 ... countk by min
else
    conclude element doesn't exist
```

**Comment by tylertreat** [comment]
Feel free to make a PR.

**Comment by tylertreat** [comment]
Resolved with #27.
